### PR TITLE
Potential README changes

### PR DESCRIPTION
**Pull request by rothsa**
The current README doesn't show off what puppet lint has developed into enough, and it's wide flexibility as far as how one can set or run particular checks. I condensed and moved the notes about options up. I removed the 'send_configuration' rake examples as the concept is already introduced in the first Rakefile explanation.

puppet-lint.rc now has it's own section because it's that cool.

Puppet-lint is very helpful for people new to puppet, and I remember at the time I first read it, 'Testing your manifests' threw me for a loop because I didn't yet understand the scope of the tool (By manifests, you mean modules and... repos too right?) For that reason I changed this title to 'Usage', which keeps one from getting distracted interpreting the title and on to the rest of the page.

### CODE REVIEW

**Comment by rothsa** [comment]
I wonder if implemented tests should not be in a separate file as well that is linked to directly by the README so that those interested can go there. That would make the README shorter, sweeter, more palatable ;-)

File: README.md [hunk 1]
```diff
@@ -4,23 +4,33 @@
...
 You can test a single manifest file by running
 
     puppet-lint <path to file>
 
-### Rake task
+Puppet Lint has a large number of options to modify which checks should be run.
+
+You can disable any of the checks when running the `puppet-lint` command by
+adding a `--no-<check name>-check` flag to the command.  For example, if you
+wanted to skip the 80 character check, you would run
```
Comment (rnelson0) [inline README.md]: This is now the 140 character check

File: README.md [hunk 2]
```diff
@@ -4,23 +4,33 @@
...
+```  
+    puppet-lint --no-80chars-check <path to file>
```
Comment (rnelson0) [inline README.md]: Same, 140 char

**Comment by rnelson0** [comment]
@rothsa This looks good, the readme definitely needs the love! In addition to the 80/140 char comments, there's a merge conflict now, if you could rebase it, please. Thank you!

**Comment by rothsa** [comment]
Hopefully looks a little better now :)

**Pull request merged**